### PR TITLE
Made the readme file in standard

### PR DESCRIPTION
**Pull request by sivaramya**
change the readme file based on the standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,70 @@
 # siddhi-map-xml
```
Comment (dilini-muthumala) [inline README.md]: This should be 

siddhi-map-xml
======================================

Please compare with https://raw.githubusercontent.com/wso2-extensions/siddhi-execution-unique/master/README.md

File: README.md [hunk 2]
```diff
@@ -1,8 +1,70 @@
...
+* <a target="_blank" href="https://wso2-extensions.github.io/siddhi-map-xml/api/4.0.0-M11-SNAPSHOT/#xml-source-mapper">xml</a> *(<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#source-mappers">Source Mapper</a>)*<br><div style="padding-left: 1em;"><p>This mapper converts XML input to Siddhi event. Transports which accepts XML messages can utilize this extension to convert the incoming XML message to Siddhi event. Users can either send a pre-defined XML format where event conversion will happen without any configs or can use xpath to map from a custom XML message.</p></div>
+
+## How to Contribute
+ 
+  * Please report issues at <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-xml/issues">GitHub Issue Tracker</a>.
+  * Send your bug fixes pull requests to <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-xml/tree/master">master branch</a>.
```
Comment (dilini-muthumala) [inline README.md]: This should be 'Send your contributions as pull requests to'...

File: README.md (hunk 1 above)
Comment (sivaramya) [reply README.md]: fixed in c8170fb

File: README.md (hunk 2 above)
Comment (sivaramya) [reply README.md]: fixed in c8170fb

**Pull request merged**